packages/components/package.json

```diff
@@ -17,7 +16,9 @@
         "./radio-group": "./dist/shadcn/components/ui/radio-group.jsx",
         "./select": "./dist/shadcn/components/ui/select.jsx",
         "./textarea": "./dist/shadcn/components/ui/textarea.jsx",
-        "./utils": "./dist/shadcn/utils.js"
+        "./utils": "./dist/shadcn/utils.js",
+        "./use-scroll-to-id": "./dist/hooks/useScrollToId.js",
```

Comment:
nit: i would feel a little better about the hooks being exported at `./hooks/*` instead of being mixed in w/ the components. but i won't block on it.

Reply:
done.